CONTRIBUTORS.md

```diff
@@ -1,6 +1,6 @@
m/jklippel), [jannic](https://github.com/jannic), [Foso](https://github.com/Foso), [JonOfUs](https://github.com/JonOfUs), [CreamyCookie](https://github.com/CreamyCookie), [Kaligule](https://github.com/Kaligule), [kvithayathil](https://github.com/kvithayathil), [luiscruz](https://github.com/luiscruz), [MStrecke](https://github.com/MStrecke), [mlasson](https://github.com/mlasson), [schwedenmut](https://github.com/schwedenmut), [M-arcel](https://github.com/M-arcel), [mgborowiec](https://github.com/mgborowiec), [mo](https://github.com/mo), [mdeveloper20](https://github.com/mdeveloper20), [Mchoi8](https://github.com/Mchoi8), [Gaffen](https://github.com/Gaffen), [mschuetz](https://github.com/mschuetz), [max-wittig](https://github.com/max-wittig), [Mengshi24](https://github.com/Mengshi24), [MolarAmbiguity](https://github.com/MolarAmbiguity), [mounirlamouri](https://github.com/mounirlamouri), [nicoolasj](https://github.com/nicoolasj), [nikhil097](https://github.com/nikhil097), [nproth](https://github.com/nproth), [oliver](https://github.com/oliver), [panoreak](https://github.com/panoreak), [patrickjkennedy](https://github.com/patrickjkennedy), [pganssle](https://github.com/pganssle), [ortylp](https://github.com/ortylp), [RafaelBod](https://github.com/RafaelBod), [ramzan](https://github.com/ramzan), [iamrichR](https://github.com/iamrichR), [SamWhited](https://github.com/SamWhited), [SebiderSushi](https://github.com/SebiderSushi), [selivan](https://github.com/selivan), [sonnayasomnambula](https://github.com/sonnayasomnambula), [sethoscope](https://github.com/sethoscope), [shantanahardy](https://github.com/shantanahardy), [shombando](https://github.com/shombando), [Silverwarriorin](https://github.com/Silverwarriorin), [danners](https://github.com/danners), [corecode](https://github.com/corecode), [vimsick](https://github.com/vimsick), [lyallemma](https://github.com/lyallemma), [edent](https://github.com/edent), [atrus6](https://github.com/atrus6), [Toover](https://github.com/Toover), [heyyviv](https://github.com/heyyviv), [waylife](https://github.com/waylife), [yarons](https://github.com/yarons), [agibault](https://github.com/agibault), [amhokies](https://github.com/amhokies), [andrewc1](https://github.com/andrewc1), [axq](https://github.com/axq), [chrk2205](https://github.com/chrk2205), [e-t-l](https://github.com/e-t-l), [fossterer](https://github.com/fossterer), [sak96](https://github.com/sak96), [gregoryjtom](https://github.com/gregoryjtom), [lightonflux](https://github.com/lightonflux), [loucasal](https://github.com/loucasal), [minusf](https://github.com/minusf), [NWuensche](https://github.com/NWuensche), [rubenh-be](https://github.com/rubenh-be), [s3lph](https://github.com/s3lph), [silansuslu](https://github.com/silansuslu), [struggggle](https://github.com/struggggle), [tamizh138](https://github.com/tamizh138), [thomasdomingos](https://github.com/thomasdomingos), [trevortabaka](https://github.com/trevortabaka), [zawad2221](https://github.com/zawad2221)
```

Comment:
This is updated using a script on the next release. Please don't change it manually.